MAINTAINERS.md

```diff
@@ -6,29 +6,30 @@ The OpenSSF Scorecard project is governed by a Steering Committee, whose operati
 
 The current standing Steering Committee members are as follows:
 
-- Stephen Augustus ([@justaugustus](https://github.com/justaugustus)), Cisco
+- Stephen Augustus ([@justaugustus](https://github.com/justaugustus)), Bloomberg L.P.
 - Raghav Kaul ([@raghavkaul](https://github.com/raghavkaul)), Google
 - Spencer Schrock ([@spencerschrock](https://github.com/spencerschrock)), Google
-- Laurent Simon ([@laurentsimon](https://github.com/laurentsimon)), Independent
-- Naveen Srinivasan ([@naveensrinivasan](https://github.com/naveensrinivasan)), Independent
 - Jeff Mendoza ([@jeffmendoza](https://github.com/jeffmendoza)), Kusari
 
 ## `scorecard-maintainers`
 
-- Stephen Augustus ([@justaugustus](https://github.com/justaugustus)), Cisco
+- Stephen Augustus ([@justaugustus](https://github.com/justaugustus)), Bloomberg L.P.
 - Raghav Kaul ([@raghavkaul](https://github.com/raghavkaul)), Google
 - Spencer Schrock ([@spencerschrock](https://github.com/spencerschrock)), Google
-- Azeem Shaikh [@azeemshaikh38](https://github.com/azeemshaikh38), Google
-- Laurent Simon ([@laurentsimon](https://github.com/laurentsimon)), Independent
-- Naveen Srinivasan ([@naveensrinivasan](https://github.com/naveensrinivasan)), Independent
 
 ## `scorecard-doc-maintainers`
 
-- Kara Olive ([@olivekl](https://github.com/olivekl)), Google
+- 
 
 ## Emeritus
 
 Former maintainers are listed here.
 Thanks for your contributions to Scorecard!
 
-- 
+- Azeem Shaikh ([@azeemshaikh38](https://github.com/azeemshaikh38))
+- Laurent Simon ([@laurentsimon](https://github.com/laurentsimon))
+- Naveen Srinivasan ([@naveensrinivasan](https://github.com/naveensrinivasan))
+
+### Docs maintainers
+
+- Kara Olive ([@olivekl](https://github.com/olivekl))
```

Comment:
Kara lost access to her original account's 2FA and is effectively locked out. I confirmed her new GitHub username with her through internal channels.
```suggestion
- Kara Olive ([@klolive](https://github.com/klolive)) (historically [@olivekl](https://github.com/olivekl))
```

Reply:
Sorry to butt in here, but if Kara has access to an SSH key that is attached to her account or a device that was previously logged in using her old account, she may be able to recover it by contacting GitHub support: https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#recovering-without-your-password-or-two-factor-authentication-credentials

Reply:
Great share; thanks Jamie!!